dataactvalidator/scripts/loaderUtils.py

```diff
@@ -38,11 +44,11 @@ def trim_item(item):
     return item
 
 
-def clean_data(data, model, field_map, field_options):
+def clean_data(data, model, field_map, field_options, required_values=[], return_dropped_count=False):
+
     """ Cleans up a dataframe that contains domain values.
 
-    Parameters:
-    ----------
+    Args:
         data : dataframe of domain values
```

Comment:
There's a space after `data` here. Depending on the editor it might not recognize that as one of the args. Also, we're missing `required_values` and `return_dropped_count` in this list.